### PR TITLE
Removed dependency tarampampam/wrappers-php

### PR DESCRIPTION
**Pull request by malyg1n**
## Description

Removed dependency `tarampampam/wrappers-php` because this package was deprecated and removed

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I wrote unit tests for my code _(if tests is required for my changes)_
- [x] I have made changes in `CHANGELOG.md` file

### Removed

Dependency `tarampampam/wrappers-php` because this package was deprecated and removed

### CODE REVIEW

File: tests/Mocks/TypedIDEntityMock.php [hunk 1]
```diff
@@ -125,6 +124,6 @@ public function toArray(): array
      */
     public function toJson($options = 0): string
```
Comment (Zaboday) [inline tests/Mocks/TypedIDEntityMock.php:125]: Добавь пожалуйста тест что если в `toJson($options)`  передать не `int` то будет выброшен TypeError.

Comment (malyg1n) [reply tests/Mocks/TypedIDEntityMock.php:125]: done

**Review by Zaboday — APPROVED** [review]

File: tests/Types/AbstractIDEntityTestCase.php [hunk 2]
```diff
@@ -135,6 +135,19 @@ public function testToJson(): void
...
+    /**
+     * @return void
+     */
+    public function testToJsonWithExceptions(): void
+    {
+        $entity = $this->entityFactory();
+        $this->expectException(\TypeError::class);
+        $entity->toJson([]);
+
+        $this->expectException(\TypeError::class);
```
Comment (eldario) [inline tests/Types/AbstractIDEntityTestCase.php]: Этот ассерт не пройдёт

**Review by eldario — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog][keepachangelog] and this project adheres to [Semantic Versioning][semver].
 
+## Unreleased
+
+### Removed
+
+Dependency `tarampampam/wrappers-php` because this package was deprecated and removed
```
Comment (eldario) [inline CHANGELOG.md]:
```suggestion
- Dependency `tarampampam/wrappers-php` because this package was deprecated and removed
```

**Pull request merged**